backend/src/controller/plantings.rs

```diff
@@ -104,7 +104,7 @@ pub async fn create(new_seed_json: Json<NewPlantingDto>, pool: Data<Pool>) -> Re
         ("oauth2" = [])
     )
 )]
-#[patch("/{id}")]
+#[put("/{id}")]
```

Comment:
Why not PATCH?
The actions should be as fine-grained as possible, e.g. the move action should only change x and y of a planting. 
Under the hood, it calls the update endpoint with patch, sending just
```json
{ "x": 123, "y": 123 }
```
These are the semantics of PATCH as I understand it.

PUT means that the whole dto would have to be sent. Making it impossible to undo/redo fine-grained changes.

Reply:
If we use the idempotent subset of PATCH it is fine, too. The way you describe it, it is perfectly okay. And I agree, when the DTOs become bigger (like Plant) we definitely want to only send minimal updates.

Do we already know how the actions in the state are described? Will they be like the PATCH example you describe here?

Reply:
> Do we already know how the actions in the state are described? Will they be like the PATCH example you describe here?
>
Yes, they look quite similar.
Though the types can not be shared.
An action in the frontend also encapsulates the behavior: 
- how to apply itself to the state.
- how to construct its inverse action.
- how to communicate itself to the backend.

Reply:
Yes, I see that there is more information in the state but what about having the shared DTO as sub-object?